nixos/jormungandr-monitor/monitor.py

```diff
@@ -53,7 +53,11 @@ def counts_gauge(addr):
 @JORMUNGANDR_METRICS_REQUEST_TIME.time()
 def process_jormungandr_metrics():
     metrics = jcli_rest(['node', 'stats', 'get'])
-    metrics['lastBlockTime'] = parse(metrics['lastBlockTime']).timestamp()
+    try:
+        metrics['lastBlockTime'] = parse(metrics['lastBlockTime']).timestamp()
+    except:
+        metrics['lastBlockTime'] = False
+        print("failed to parse lastBlockTime")
```

Comment:
i would maybe try to print what its failing to parse, so you can debug things if its a new failure